yt_dlp/extractor/iwara.py

```diff
@@ -46,14 +46,17 @@ def _get_media_token(self, invalidate=False):
         if not invalidate and self._MEDIATOKEN:
             return self._MEDIATOKEN
 
```

Comment:
```suggestion
        if not invalidate and self._MEDIATOKEN:
            return self._MEDIATOKEN
        elif invalidate:
            IwaraBaseIE._MEDIATOKEN = None
```
or else an expired(?) token will still be returned if the exception below is caught